java/sage/IOUtils.java

```diff
@@ -18,12 +18,65 @@
 import sage.io.RemoteSageFile;
 import sage.io.SageInputStream;
 
+import java.io.BufferedReader;
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+
 public class IOUtils
 {
   private IOUtils()
   {
   }
 
+  /**
+   * Used to read a URL contents into String.  This operation blocks and the content is a string, so it should
+   * not be used on large files, and should not be used on binary files.
+   *
+   * @param urlStr URL to download
+   * @return String contents of the given URL
+     */
+  public static String getUrlAsString(String urlStr)
+  {
+    StringBuilder sb = new StringBuilder();
+    try
+    {
+      URL url = new URL(urlStr);
+      InputStream is = url.openStream();
+      BufferedReader br = new BufferedReader(new InputStreamReader(is));
+
+      String line;
+      while ( (line = br.readLine()) != null)
```

Comment:
That's a good idea... i did look at fileToString but it seemed overly complex :)  I could have used a byte reader, but this method was really to read into a string.  I'll re-look at the file to string again, and see what I can do.